### DISCUSSION TITLE
Do not trigger (refresh) when typing with autoTrigger disabled.

### DISCUSSION THREAD

**Discussion by amgodoi** [neoclide/coc.nvim#3976] [Q&A] [answered]
Hi!

I'm using autoTrigger like this:
    "suggest.autoTrigger": "none",

After type <c-space> the popup autoTrigger window opens (as expected), but not refresh or trigger anymore. I have to press <c-space> again to trigger/refresh one more time and so on. Appears that worked, but is does not working anymore.

Is there any way to anable autotrigger/refresh on type only when autoTrigger window is already open?

**Comment by chemzqm** [comment]
The completion should be not from coc.nvim or you have triggered `coc#refresh()`, it's expected behavior.

**Reply by amgodoi** [reply]
Hi!

With autoTrigger disabled could I use coc#refresh() in some way? Maybe mapping something?

**Reply by chemzqm** [reply]
Yes

**Reply by amgodoi** [reply]
How could i do that? Some example, please? I did not found anything from documentation.

**Comment by chemzqm** [comment, marked as answer]
Fixed on latest release branch of coc.nvim.

`:h coc#refresh()`
